js/src/features/project/types.d.ts

```diff
@@ -72,4 +72,6 @@ declare module "ProjectModels" {
     pageSize?: number;
     currentPage?: number;
   }
+  
+  export type RolePermissions = string[]
```

Comment:
You can include this type into the `Role` type above, can't you?

Reply:
done